pkg/machines/libvirt-dbus.es6

```diff
@@ -268,6 +270,31 @@ LIBVIRT_DBUS_PROVIDER = {
                 });
     },
 
+    CREATE_STORAGE_POOL({
+        connectionName,
+        name,
+        type,
+        source,
+        target,
+        host,
+        autostart,
+    }) {
+        const poolXmlDesc = getPoolXML({name, type, source, target, host});
+        let storagePoolPath;
+
+        return (dispatch) => call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'StoragePoolDefineXML', [poolXmlDesc, 0], TIMEOUT)
+                .then(poolPath => {
+                    storagePoolPath = poolPath;
+                    return call(connectionName, storagePoolPath[0], 'org.libvirt.StoragePool', 'Create', [Enum.VIR_STORAGE_POOL_CREATE_NORMAL], TIMEOUT);
```

Comment:
Sounds a little edge-casey to me.
Why wouldn't you just start the NFS server first?

Reply:
I understand the usecase but it's indeed kinda edgy and I would rather not flood the modal with 'not-clear-what-they-do' options to support such scenarios.
At least until the need appears.

Reply:
You might not be connected to the correct network at the moment, it's not always the case that the NFS server is not started, it might be running, that's why I used "not accessible".  This would mimic the same logic as we have for domains, you can also define domain without starting it.  I agree that it will not be used that often, just thought that it would be nice feature to have.

Reply:
Let's add the basic functionality first and we can consider such advanced options for future RFE.

Reply:
Sure, I'm OK with that.